ospfd/ospf_zebra.c

```diff
@@ -478,7 +478,7 @@ void ospf_zebra_delete(struct ospf *ospf, struct prefix_ipv4 *p,
 	if (IS_DEBUG_OSPF(zebra, ZEBRA_REDISTRIBUTE)) {
 		char buf[INET_ADDRSTRLEN];
 		zlog_debug("Zebra: Route delete %s/%d",
-			   inet_ntop(AF_INET, &p->prefix, buf, sizeof(buf[0])),
```

Comment:
Why not switch to prefix2str?

Reply:
Happy to do that - there is still some 'ntop' out there, I was just doing the minimal thing. but even prefix2str needs to see the right buffer size... Would you rather see a slightly larger diff that replaces the ntop use in ospf_zebra generally, or just this one instance?

Reply:
Sure - it's only a couple of places. I will update the PR...